File: algorithm/boj/_1000/1978.py
import sys

if __name__ == "__main__":
    n = int(sys.stdin.readline())
    numbers = list(map(int, sys.stdin.readline().split()))
    cnt = 0
    for i in range(n): # 0~5
        if numbers[i]==1: continue
        for j in range(2, numbers[i]): # 2~9
            if numbers[i]%j==0:
                cnt -=1 ; break
        cnt += 1
    print(cnt)

File: algorithm/boj/_1000/1978.c
#include <stdio.h>

int main() {
    int n, u, i, j, cnt = 0;
    scanf("%d", &n);
    
    for(i=0; i<n; i++) {
        scanf("%d", &u);
        if(u==1) continue;
        for(j=2; j<u; j++) {
            if(u%j==0) {
                cnt--;
                break;
            }
        }
        cnt++;
    }
    printf("%d\n", cnt);
}